assets/js/modules/idea-hub/components/dashboard/DashboardIdeasWidget/index.stories.js

```diff
@@ -235,6 +240,44 @@ DataUnavailableAll.decorators = [
 	},
 ];
 
+export const Activity = Template.bind( {} );
+Activity.storyName = 'Activity In Progress';
+Activity.decorators = [
+	( Story ) => {
+		const setupRegistry = ( registry ) => {
+			registry
+				.dispatch( MODULES_IDEA_HUB )
+				.setActivity(
+					'ideas/17450692223393508734',
+					IDEA_HUB_ACTIVITY_IS_DELETING
+				);
+			registry
+				.dispatch( MODULES_IDEA_HUB )
+				.setActivity(
+					'ideas/14025103994557865535',
+					IDEA_HUB_ACTIVITY_IS_PINNING
+				);
+			registry
+				.dispatch( MODULES_IDEA_HUB )
+				.setActivity(
+					'ideas/7612031899179595408',
+					IDEA_HUB_ACTIVITY_CREATING_DRAFT
+				);
+		};
+
+		enabledFeatures.clear();
```

Comment:
This shouldn't be necessary as enabled features should be cleared before every story automatically 🙂

Reply:
Thanks for catching that @aaemnnosttv . I updated all the stories.